padertorch/contrib/data/wsj0_mix/prepare_data.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Put your paths here!
+
+# Path to the database as generated by the matlab scripts
+database_path=
+
+# If you need transcriptions, put the path to the WSJ root folder here
+wsj0_root=
+
+python -m padertorch.contrib.data.wsj0_mix.create_json \
+	--database_path ${database_path} \
+	--wsj0_root ${wsj0_root} \
+	--json_path wsj0_mix_8k.json \
```

Comment:
Why do you exclude `min` in the json name?

Reply:
For no reason. I can include it